Components/FortuneMessage.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState, useEffect } from 'react'
+import styled from 'styled-components/native'
+
+const FortuneText = styled.Text`
+font-size: 30px;
+color: white;
+margin-bottom: 20px;
+text-align: center;
+`
+const fortuneURL = "https://type.fit/api/quotes"
+
+export const FortuneMessage = () => {
+        const [fortune, setFortune] = useState("")
+
+        const getRandomInt = (max) => {
+            return Math.floor(Math.random() * Math.floor(max))
+        }
+
+        const randomizeFortune = () => {
+            fetch(fortuneURL)
+                .then(response => response.json())
+                .then(json => {
+
+        const formattedArray = json.map(item => item.text)  
+        const randomInt = getRandomInt(formattedArray.length)
```

Comment:
Wrong indentations as well

Reply:
I've made a change in the code, however I'm not sure it's how you meant

Reply:
It's ok now ;)